tests/unit_tests/pyanaconda_tests/modules/subscription/test_subscription.py

```diff
@@ -1065,21 +1053,26 @@ def test_register_and_subscribe(self, publisher):
         task_path = self.subscription_interface.RegisterAndSubscribeWithTask()
         obj = check_task_creation(self, task_path, publisher, RegisterAndSubscribeTask)
         # check all the data got propagated to the task correctly
-        self.assertEqual(obj.implementation._rhsm_observer, rhsm_observer)
-        self.assertEqual(obj.implementation._subscription_request, subscription_request)
-        self.assertEqual(obj.implementation._system_purpose_data, system_purpose_data)
-        self.assertEqual(obj.implementation._registered_callback,
-                         self.subscription_module.set_registered)
-        self.assertEqual(obj.implementation._registered_to_satellite_callback,
-                         self.subscription_module.set_registered_to_satellite)
-        self.assertEqual(obj.implementation._subscription_attached_callback,
-                         self.subscription_module.set_subscription_attached)
-        self.assertEqual(obj.implementation._subscription_data_callback,
-                         self.subscription_module._set_system_subscription_data)
-        self.assertEqual(obj.implementation._satellite_script_downloaded_callback,
-                         self.subscription_module._set_satellite_provisioning_script)
-        self.assertEqual(obj.implementation._config_backup_callback,
-                         self.subscription_module._set_pre_satellite_rhsm_conf_snapshot)
+        assert obj.implementation._rhsm_observer == rhsm_observer
+        assert obj.implementation._subscription_request == subscription_request
+        assert obj.implementation._system_purpose_data == system_purpose_data
+        # pylint: disable=comparison-with-callable
+        assert obj.implementation._registered_callback == self.subscription_module.set_registered
+        # pylint: disable=comparison-with-callable
+        assert obj.implementation._registered_to_satellite_callback == \
+            self.subscription_module.set_registered_to_satellite
+        # pylint: disable=comparison-with-callable
+        assert obj.implementation._subscription_attached_callback == \
+            self.subscription_module.set_subscription_attached
+        # pylint: disable=comparison-with-callable
+        assert obj.implementation._subscription_data_callback == \
+            self.subscription_module._set_system_subscription_data
+        # pylint: disable=comparison-with-callable
+        assert obj.implementation._satellite_script_downloaded_callback == \
+            self.subscription_module._set_satellite_provisioning_script
+        # pylint: disable=comparison-with-callable
+        assert obj.implementation._config_backup_callback == \
+            self.subscription_module._set_pre_satellite_rhsm_conf_snapshot
```

Comment:
Hi @M4rtinK could you please confirm these pylint disables are fine here?

Reply:
Seems fine to me. While I understand why pylint has this check (to avoid people using callable instead of the value returned by executing the callable) in this case the behaviour is correct - we need to check if all the callbacks were correctly forwarded to the task, so the check needs to be turned off.